opendevin/state.py

```diff
@@ -11,10 +11,15 @@
     CmdOutputObservation,
 )
 
+
 @dataclass
 class State:
     plan: Plan
     iteration: int = 0
-    background_commands_obs: List[CmdOutputObservation] = field(default_factory=list)
+    # number of characters we have sent to and received from LLM so far for current task
+    num_of_chars: int = 0
```

Comment:
Maybe a better name would be `total_prompt_chars`

Reply:
`total_prompt_` is not correct because this counts both input chars and output chars. I do see the benefit of limiting prompts and responses separately (since output tokens are usually more expensive than input ones), but as of now I don't want to make it overcomplicated.